js/app.js

```diff
@@ -32,6 +39,21 @@
    * to the button that has an id of the same name.
    */
 
+  const btn1 = document.querySelector("#btn1");
```

Comment:
There's actually a more direct document method that will give you a element by id. This is more a suggestion than a recommended fix.